src/doc/src/commands/cargo-test.md

```diff
@@ -310,6 +310,17 @@ specified with the <code>CARGO_TARGET_DIR</code> environment variable, or the
 <code>build.target-dir</code> <a href="../reference/config.html">config value</a>.
 Defaults to <code>target</code> in the root of the workspace.</dd>
 
+    
+ 
+If we want to see printed values for passing tests as well, we can tell Rust
+to also show the output of successful tests at the end with `--show-output`.
+
+```console
+$ cargo test -- --show-output
+```
+
+When we run the tests in Listing 11-10 again with the `--show-output` flag, we
+see the following output.
```

Comment:
Here are things I am concern about:

- Under the section "Display Options", we have mentioned the `--nocapture` flag, which is somehow overlapped with `--show-output`. The change you made here might be better moving there and reword a bit, though personally, I am not sure where `--show-output` is that worthy to mention.
- Throughout the whole cargo-test documentation, the pronoun "you" is more common than "we". Could you change it to "you" for consistency? 
- cargo-bench gets [a similar paragraph](https://github.com/rust-lang/cargo/blob/master/src/doc/man/cargo-bench.md#display-options) about the test harness option `--nocapture`. You would like to update it as well.